src/views/text-parsing/ui/text-parsing-result-empty.tsx

```diff
@@ -0,0 +1,11 @@
+import { Center } from '@mantine/core'
+
+export const TextParsingResultEmpty = () => {
+  return (
+    <Center h="100%">
+      <p className="text-sm text-gray-500 dark:text-gray-400">
+        텍스트를 입력하고 &apos;분석하기&apos; 버튼을 클릭하면 이곳에 분석 결과가 표시돼요
```

Comment:
[제안]
아래와 같이 개행문자로 한번 끊어주는 것이 보기에 깔끔한 것 같은데 어떠신가요? 
괜찮다고 생각하신다면 반응형 작업 때 포함 부탁드리겠습니다.

<br />

```suggestion
        텍스트를 입력하고 &apos;분석하기&apos; 버튼을 클릭하면 <br />이곳에 분석 결과가 표시돼요
```